### PR TITLE
Draft: Allow to export commands to host

### PR DESCRIPTION
**Pull request by bellegarde-c**
This allows to run containers commands from host.

### CODE REVIEW

**Comment by bellegarde-c** [comment]
Ok, lot of code to fix and some missing features:
- Allow to clear cache
- ZSH completion support (not sure how it works)

**Comment by bellegarde-c** [comment]
[Capture vidéo du 2022-10-23 15-16-21.webm](https://user-images.githubusercontent.com/57196916/197394515-49ac018b-0ade-4e42-b15b-ed45f4d5647d.webm)

File: distrobox-path [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+                ;;
+        esac
+    done
+}
+
+if [ -z "${container_name}" ] ; then
+    if [ "${self}" != "distrobox-path" ]; then
+        container_name="$(echo "$0" | awk -F '/' '{ print $(NF-1) }')"
+        # Escape args
+        args="$(printf "${1+ %q}" "$@")"
```
[the remark this answers is not in the document]
Comment (bellegarde-c) [reply distrobox-path:146]: No idea how to fix this one.

**Comment by 89luca89** [comment]
Hi @bellegarde-c 
Can you explain a bit better what is the PR about?

From what I can see, this aims to export **ALL** binaries from one distrobox to the host? 
In this case, I'm not sure if this is something I'd like in distrobox. 
Exporting one binary at time on user's choice (like it's done with distrobox-export) is very different from bulk exporting.

I'm sorry to tell this after you have already done the work

**Comment by bellegarde-c** [comment]
* It exports all commands not available on the host from a target container
* It exports bash completion from target container

In my case, using distrobox-export means exporting 445 binaries one by one.

But not an issue if not merged, I use this script since months and doing this merge request allowed me to clean up the original garbage code :)

**Comment by 89luca89** [comment]
Thanks @bellegarde-c for understanding :smile:

**Pull request closed without merging**